packages/cloudflare/src/api/do-sharded-tag-cache.ts

```diff
@@ -27,46 +34,159 @@ interface ShardedD1TagCacheOptions {
    * @default 5
    */
   regionalCacheTtlSec?: number;
+
+  /**
+   * Whether to enable shard replication
+   * Shard replication will duplicate each shards into N replicas to spread the load even more
+   * For example with N being 2, tag `tag1` could be read from 2 different durable object instance
+   * On read you only need to read from one of the shards, but on write you need to write to all shards
```

Comment:
"""
All replicas of the a shard contain the same value - write are sent to all of the replicas.
This allows most frequent read operations to be sent to only one replica to spread the load.
"""